cmd/watch.go

```diff
@@ -102,6 +109,16 @@ func handleWatchEvent(client kit.ThemeClient, asset kit.Asset, event kit.EventTy
 		blue(asset.Key),
 	)
 	if err := perform(client, asset, event, nil); err != nil {
+		notificationMessage("There is a error in your templates check your logs", "Watch", "")
 		stdErr.Printf("[%s] %s", green(client.Config.Environment), err)
 	}
 }
+
+func notificationMessage(message string, title string, iconPath string) {
+	notify = notificator.New(notificator.Options{
+		DefaultIcon: basepath + "/docs/assets/images/shopify-logo.png",
```

Comment:
How would this work with a single binary distribution. Part of the design of themekit is to be a single binary distro to make it easily portable. If notificator could take a file instance we could create a go file with zip data of an icon.

Reply:
The `DefaultIcon` was not even working for me. It was showing the default notification icon.

> How would this work with a single binary distribution

I do not know yet. I will have to investigate your suggestion about zipping the data of the icon.